ddtrace/debugging/_uploader.py

```diff
@@ -34,6 +35,13 @@ def __init__(self, encoder, interval=None):
             "Content-type": "application/json; charset=utf-8",
             "Accept": "text/plain",
         }
+
+        container_info = container.get_container_info()
```

Comment:
Should we cache the return value of [container.get_container_info()](https://github.com/P403n1x87/dd-trace-py/blob/1153e1d96f968b0bbeb8d323e34b3099e43ade52/ddtrace/internal/runtime/container.py#L83)? 

I think we access this method in a few places (AgentWriter, TelemetryWriter, Profiler) and it seems like an expensive  operation.

Reply:
Probably, worth another PR IMHO.